content/docs/howto/create_your_own_pair.md

```diff
@@ -44,5 +44,4 @@ Then, you may execute the contract with the organized JSON above.
 
 ## IMPORTANT - Only for Terra 2.0
 
-PLEASE BE AWARE that the transaction would work only if you send the tiny number (like 0.000001) of native or IBC tokens in case of creating a pair that contains them.
-The contract uses the received token to validate whether it exists or not.
+PLEASE BE AWARE that to create a native token in a pair, it must be registered in the factory before it can be used. Contact [Terraswap Discord](https://discord.gg/XERFR8dCWv) to register
```

Comment:
why dont we recommend contact action just in case of not registered token? 🤔

Reply:
Good to add `a native token(including IBC tokens)`

Reply:
> why dont we recommend contact action just in case of not registered token? 🤔

Otherwise, anyone can create a pair.

Reply:
> Good to add `a native token(including IBC tokens)`

If meaning of native token as cosmos denom, I think that ibc is also one of the native tokens.

Reply:
can't anyone create a pair contains registered native token like Luna? I thought this message is for someone who wants to create a pair using not registered one.

Reply:
sorry I misunderstood the message, forget it all I wrote above. I just hope there's a brief guide on how to check if the token is registered.

Reply:
https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/39/commits/22a16c2ba7a90a0d00da6cfc380c1d331eaa6ba1